dev-util/cmake-fedora/files/cmake-fedora-2.7.1-no-release.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index a0b8c7a..2738be0 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -148,17 +148,6 @@ ADD_CUSTOM_TARGET_COMMAND(module-only
+ # Release
+ #
+ 
+-ADD_DEPENDENCIES(upload tag_push)
```

Comment:
Does not appear to be required but if we do not get rid of this then build-time stuff that we don't need to run (and possibly should not) will run:

```       
[Off] .fedora-upload-ca.cert not found, Fedora release support disabled.                    
[Off] fedpkg not found, Fedora release support disabled.                                     
[Off] koji not found, Fedora release support disabled.                                      
[Off] bodhi not found, Fedora release support disabled.   
```

By removing these steps completely via this patch, we remove the possibility someone will file a bug because for some reason they have a command named `fedpkg` (or others) in `$PATH` and it causes a build failure.

If it is acceptable to allow this build-time check, then the patch is unnecessary. I have not found a great way to disable this another way because there's still the else condition which calls `MANAGE_RELEASE(upload tag_push)` which, if the other pre-requisite binaries are found, would try to upload a built RPM using scp.